### PR TITLE
Export tikv status port in CI

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tiflash/issues/8023

Problem Summary:

All CI failed after https://github.com/tikv/pd/pull/6945, the pd coordinator can not start because the pd can not connect to tikv status port.

### What is changed and how it works?

Export the status port of tikv in CI

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

File: tests/docker/cluster.yaml [hunk 1]
```diff
@@ -45,6 +45,7 @@ services:
     command:
       - --addr=0.0.0.0:20160
       - --advertise-addr=tikv0:20160
+      - --status_address=tikv0:20180
```
Comment (LykxSassinator) [inline tests/docker/cluster.yaml:48]:
```suggestion
      - --status-addr=tikv0:20180
```

**Review by CalvinNeo — APPROVED** [review]
lgtm

File: tests/docker/run.sh
Comment (Lloyd-Pottiger) [inline tests/docker/run.sh:1]: file permission is changed, is this as expected?

Comment (JaySon-Huang) [reply tests/docker/run.sh:1]: Yes, `chmod u+x` for it

**Review by Lloyd-Pottiger — APPROVED** [review]

**Comment by JaySon-Huang** [comment]
waiting https://ci.pingcap.net/blue/organizations/jenkins/tiflash-ghpr-unit-tests/detail/tiflash-ghpr-unit-tests/11180/pipeline

**Pull request merged**